### PR TITLE
Revert to Latest Container Image

### PR DESCRIPTION
**Pull request by mns4595**
This should be reverted once the devcontainer image is fixed

### CODE REVIEW

File: .github/workflows/cmake-embedded.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ jobs:
     # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/staflsystems/stafl-devcontainer-ci@sha256:085e82c70a1b64fa94cf21f9f7f18b255cec010869a44de73e76994dd82d66f3
+      image: ghcr.io/staflsystems/stafl-devcontainer-ci
```
Comment (mns4595) [inline .github/workflows/cmake-embedded.yml]: This should probably point to a particular tag

**Comment by harmandersihra** [comment]
lol

**Comment by mns4595** [comment]
> lol

🙃

**Review by willson556 — APPROVED** [review]

**Comment by harmandersihra** [comment]
Please remove my email from this lol

**Pull request merged**